src/components/ComingSoon.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div class="coming-soon_wrapper">
+    <div class="headings">
+      <h3>SOON IN THE CINEMA</h3>
+      <h3>SOON</h3>
+      <h3>SEE ALL</h3>
+    </div>
+    <div class="soon-movies_wrapper">
+      <MovieCard
+        title="Predator"
+        img="/src/assets/img/predator.png"
+        description="A team of commandos on a mission in a Central American jungle find themselves haunted by an extraterrestrial warrior."
+      ></MovieCard>
+      <MovieCard
+        title="Commando"
+        img="{movie2}"
```

Comment:
in Vue this
```jsx
:img="movie2"
```
works almost the same as React's
```jsx
img="{movie2}"
```
but it can't be used alternatively ;)